faster_sam/openapi.py

```diff
@@ -5,7 +5,40 @@
 
 
 def custom_openapi(app: FastAPI, openapi_schema: Dict[str, Any]) -> Callable[[], Dict[str, Any]]:
+    """
+    Custom OpenAPI generator for the FastAPI application.
+
+    Parameters
+    ----------
+    app : FastAPI
+        The FastAPI application instance.
```

Comment:
`A` no lugar de `the` para deixar claro que é qualquer instância e não uma específica?